rs/types/types/src/crypto/hash/domain_separator.rs

```diff
@@ -113,6 +114,7 @@ impl DomainSeparator {
             DomainSeparator::IDkgTranscript => "ic-idkg-transcript-domain",
             DomainSeparator::EcdsaSigShare => "ic-threshold-ecdsa-sig-share-domain",
             DomainSeparator::SchnorrSigShare => "ic-threshold-schnorr-sig-share-domain",
+            DomainSeparator::VetKdShare => "ic-vetkd-share-domain",
```

Comment:
In my next PR I'll introduce `ic_types::crypto::vetkd::VetKdEncryptedKeyShareContent` and make that the type of `ic_types::crypto::vetkd::VetKdEncryptedKeyShare::encrypted_key_share` so that I can specify a `DomainSeparator` for it (so that I can sign it and create the `node_signature`). Because I was thinking to use `ic-vetkd-encrypted-key-share-content-domain` as domain separator string, we could consider using `ic-vetkd-encrypted-key-share-domain` as domain string here. But that somewhat depends on which name we  actually end up using for the struct (see my other comment), and it would also be OK if they are not perfectly in line.

Reply:
Changed to `"ic-vetkd-key-share-domain"` for now

Reply:
👍